Example/MoyaComparison/MoyaComparison/Models/Realm/Video.swift

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright 2017 niji
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import RealmSwift
+import Realm
+
+final class Video: Object, DualJSONConformance {
+  
+  enum Attributes: String {
+    case identifier = "id"
```

Comment:
Prefer using `static let` for your constants here (because otherwise you will never use the `case` directly but always have to call `.rawValue`…). An `enum` with _no_ `case` might seem strange, but it's commonly used as a "namespacing" workaround